man/start-stop-daemon.8

```diff
@@ -164,6 +164,12 @@ log it or send it to another location.
 .It Fl -capabilities Ar cap-list
 Start the daemon with the listed inheritable, ambient and bounding capabilities.
 The format is the same as in cap_iab(3).
+.It Fl -secbits Ar sec-bits
```

Comment:
i was going to complain about the name & argument format, but this seems to match the `capsh` interface, so OK